### DISCUSSION TITLE
schema.org:Issue & schema.org:PullRequest for GitHub, GitLab, JIRA et al.

### DISCUSSION THREAD

**Discussion by kael** [schemaorg/schemaorg#3109] [Ideas]
Wondering if there's any plan to implement specific schemas  `http://schema.org/Issue` & `http://schema.org/PullRequest` for GitHub-like issue and JIRA tickets, and also for pull requests.

For example `http://schema.org/SoftwareSourceCode` is used for pages of GitHub projects, even for discussions forum which should include `https://schema.org/DiscussionForumPosting` instead, and GitHub gists include  `http://schema.org/Code`.

Having more specific schemas would be nice.

**Comment by thadguidry** [comment]
We don't have Issue and likely not needed because you can frame up your own quite easily by just using CreativeWork as the main type and give a more specific type in additionalType and use a Wikidata identifier for Issue like perhaps https://www.wikidata.org/wiki/Q11684032 .  And liberally use Comment, Question, Answer types and their properties along with the various subproperties available as seen at bottom of https://schema.org/Comment
